### DISCUSSION TITLE
Browser compatibility issue

### DISCUSSION THREAD

**Discussion by Amsiam** [saadeghi/daisyui#2248] [Q&A]
I am facing some browser compatibility issues in Collapse..?  Collapse doesn't open in some browsers.  Does anyone know how to fix this?

**Comment by saadeghi** [comment]
Which browser and which version?

**Reply by Amsiam** [reply]
Safari browser. :)

**Reply by saadeghi** [reply]
Safari:

https://github.com/saadeghi/daisyui/assets/7342023/c90086c5-5d95-4f63-b9da-907834efcc97

**Reply by Amsiam** [reply]
Problem in  mobile browser.

**Reply by saadeghi** [reply]
Safari mobile:

https://github.com/saadeghi/daisyui/assets/7342023/72297979-c4e4-4f14-901e-2c8d5ae9c722
